### PR TITLE
Set Policy For Groups

### PR DESCRIPTION
**Pull request by dvaldivia**
Also prettier 2 on some files 

Screen Shot 2020-05-22 at 12 51 36 PM

### CODE REVIEW

File: portal-ui/src/screens/Console/Policies/SetPolicy.tsx [hunk 1]
```diff
@@ -101,12 +112,13 @@ const SetPolicy = ({
   };
 
   useEffect(() => {
+    console.log("im openx", open);
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/SetPolicy.tsx:115]: Can we remove these console logs please?
```suggestion
```

File: portal-ui/src/screens/Console/Policies/SetPolicy.tsx [hunk 2]
```diff
@@ -101,12 +112,13 @@ const SetPolicy = ({
...
     if (open) {
       console.log("im open");
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/SetPolicy.tsx:117]:
```suggestion
```

File: portal-ui/src/screens/Console/Policies/SetPolicy.tsx [hunk 3]
```diff
@@ -101,12 +112,13 @@ const SetPolicy = ({
...
       console.log(selectedUser);
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/SetPolicy.tsx:118]:
```suggestion
```

**Comment by Alevsk** [comment]
LGTM

**Pull request merged**